Core/include/Acts/Seeding/SeedfinderConfig.hpp

```diff
@@ -150,6 +159,25 @@ struct SeedfinderConfig {
   int nTrplPerSpBLimit = 100;
   int nAvgTrplPerSpBLimit = 2;
 
+  // Delegates for accessors to detailed information on double measurement that
+  // produced the space point.
+  // This is mainly referring to space points produced when combining
+  // measurement from strips on back-to-back modules.
+  // Enables setting of the following delegates.
+  bool useDetailedDoubleMeasurementInfo = false;
+  // Returns half of the length of the top strip.
+  Delegate<float(const SpacePoint*)> getTopHalfStripLength;
+  // Returns half of the length of the bottom strip.
+  Delegate<float(const SpacePoint*)> getBottomHalfStripLength;
+  // Returns direction of the top strip.
+  Delegate<Acts::Vector3(const SpacePoint*)> getTopStripDirection;
+  // Returns direction of the bottom strip.
+  Delegate<Acts::Vector3(const SpacePoint*)> getBottomStripDirection;
+  // Returns distance between the centers of the two strips.
+  Delegate<Acts::Vector3(const SpacePoint*)> getStripCenterDistance;
+  // Returns position of the center of the bottom strip.
+  Delegate<Acts::Vector3(const SpacePoint*)> getBottomStripCenterPosition;
```

Comment:
Stylistic comment: if the argument is never supposed to be `nullptr`, using a reference is clearer.

Reply:
done!